### DISCUSSION TITLE
Integration tests for syslog

### DISCUSSION THREAD

**Discussion by melezhik** [nikoksr/notify#466] [Q&A]
Hi guys! Do you need integration tests? I've written an integration test for syslog and alpine linux -  https://ci.sparrowhub.io/report/1811 , please let me know if you're interested ...

**Comment by nikoksr** [comment]
Hi @melezhik, wow, such a cool idea! Great work. We love seeing notify being used in ways, we didn't predict. Thank you so much for sharing this!

The idea itself is great and would definitely improve the project overall, I think however, that we'd need to do this for all/most services to be a sustainable option for us. I hope you understand and thank you so much again!
